src/components/UserInfo/UserInfo.scss

```diff
@@ -0,0 +1,5 @@
+@import "./../../utils/variables.scss";
+
+.UserInfo {
+  color: $-color-text-gray;
```

Comment:
Consider using a more descriptive variable name instead of `$-color-text-gray`. For example, `$text-color-secondary`. 🎨